google/cloud/storage/tools/run_emulator_utils.sh

```diff
@@ -111,14 +104,12 @@ start_emulator() {
   grpc_port=$(curl -s --retry 5 --retry-max-time 40 "http://localhost:${emulator_port}/start_grpc?port=${port}")
 
   if [ "${grpc_port}" -eq "${grpc_port}" ] 2>/dev/null; then
-    echo "Successfully connected to gRPC server at port ${grpc_port}"
+    io::log_green "Successfully connected to gRPC server at port ${grpc_port}"
   else
     echo "${IO_COLOR_RED}${grpc_port} must be an integer" >&2
     echo "${IO_COLOR_RED}Cannot connect to gRPC server; aborting test.${IO_RESET}" >&2
     cat gcs_emulator.log
```

Comment:
I'm not quite sure why we need any of the `>&2`s, but we're at least inconsistent in whether the `gcs_emulator.log` goes there too.

Reply:
You are right, removed all the `>&2`.